### DISCUSSION TITLE
Same PMS but between users

### DISCUSSION THREAD

**Discussion by Minxster** [luigi311/JellyPlex-Watched#111] [Q&A] [answered]
I'm probably answering my own question, but is there a way to have one PMS but use this to transfer watch history between user accounts? 

I have a Plex Home user, but they cant have access to a watchlist without having their own dedicated login account. So I'm planning on migrating the user data. I think I could setup a dedicated docker container just for this job, but using different "names" for my single PMS? E.g. http://10.0.10.10:32400 and then http://ubuntu-pms:32400, which are both the "same server".

Does this seem like a good idea, or is there something baked in that I can use to accomplish this?

**Comment by Minxster** [comment]
I did try this using different server names, for the same server, but connections were being refused. Probably because it's the same server 🤣

Anywho, I think I "over thought" this, I could simple use WebTools-NG for this specific issue 🤷‍♂️😉

I'd still be interested to hear your thoughts on this 😊

**Comment by luigi311** [comment, marked as answer]
Im not doing any filtering so you should be able to just do "http://10.0.10.10:32400,http://10.0.10.10:32400" for your servers and use the same api key. You then just have to add user mapping between the two accounts you want to sync.
